docs/guides/lending.md

```diff
@@ -0,0 +1,107 @@
+
+# Lend and Borrow
+
+The lending protocol allows users to trustlessly lend and borrow. Assets that are supplied to the lending pool start earning interest immediately, and can be used as a security deposit for over-collateralized borrowing. Borrowers can repay their loan at any time, along with the interest they owe, while lenders can only withdraw their deposit if there is more liquidity left in the pool (i.e. not borrowed) than the amount to be withdrawn. Interest rates are variable and depend on the supply and demand of each asset.
+
+At the end of this guide you will have:
+
+- [x] [Supplied an asset and started earning interest](#_3-deposit-to-a-lending-market)
+- [x] [Enabled the deposit as borrow collateral](#_4-enable-the-collateral-toggle)
+- [x] [Borrowed using the collateral](#_5-borrow)
+- [x] [Repaid a loan](#_6-repay-a-loan)
+- [x] [Withdrawn part of the supplied amount](#_7-withdraw-a-deposit)
+
+## Prerequisites
+
+Make sure you have a compatible [wallet plugin](guides/wallets-explorers.md#substrate-wallets) installed.
+
+## Lend and Borrow
+
+### 1. Go to the lending page
+
+<!-- tabs:start -->
+#### **Interlay**
+
+Coming soon!
+
+#### **Kintsugi**
+
+Coming soon!
+
+#### **Testnet-Kintsugi**
+
+[kintnet.interlay.io/lending](https://kintnet.interlay.io/lending)
+
+#### **Testnet-Interlay**
+
+[testnet.interlay.io/lending](https://testnet.interlay.io/lending)
+<!-- tabs:end -->
+
+There are three markets available in the screenshot attached. You can take a look at the various interest rates and decide which ones you are interested in supplying or borrowing.
+
+![The Lending page](../_assets/img/guide/lending-page-initial.png)
+
+### 2. Obtain funds to pay the transaction fees and lend
+
+You will need some of the native on-chain currency (KINT on Kintsugi, INTR on Interlay) to pay the transaction fee. Additionally, to prevent spam, you will need to place a small deposit which will be returned to you once the request has been completed.
```

Comment:
> Additionally, to prevent spam, you will need to place a small deposit which will be returned to you once the request has been completed.

Do we have griefing collateral on the lending protocol?